### PR TITLE
Fix links in the tutorial to other parts of the documentation.

### PR DESCRIPTION
**Pull request by laurin-no**
Hi there.
I did the tutorial and I saw that some internal links to other parts of the documentation were broken, so it fixed them quickly. Please review and let me know, if there are any improvements needed.

Laurin

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -1,5 +1,5 @@
 source "https://rubygems.org"
-ruby '2.4.4'
+ruby '2.5.1'
```
Comment (sjrd) [inline Gemfile]: Could you revert those changes to the version of Ruby, please? They don't make sense as part of this PR, and they seem to be breaking the build anyway.

File: Gemfile.lock [hunk 2]
```diff
@@ -83,7 +83,7 @@ DEPENDENCIES
   uglifier (= 2.7.2)
 
 RUBY VERSION
-   ruby 2.4.4p296
+   ruby 2.5.1p57
```
Comment (sjrd) [inline Gemfile.lock]: Same.

**Comment by laurin-no** [comment]
I reverted the changes but the build still fails with the following message: `/home/travis/.rvm/rubies/ruby-2.4.4/lib/ruby/site_ruby/2.4.0/rubygems.rb:289:in `find_spec_for_exe': can't find gem bundler (>= 0.a) with executable bundler (Gem::GemNotFoundException)`

Did you experience this problem before?

**Comment by sjrd** [comment]
I did not experience that problem before. I will investigate in the coming days.

**Comment by myyk** [comment]
I started a similar fix after working through the tutorial. This change looks good to me.

**Comment by laurin-no** [comment]
> I did not experience that problem before. I will investigate in the coming days.

Are there any updates?

**Pull request merged**